etc/production.ini

```diff
@@ -21,8 +21,8 @@ pyramid_force_https.structlog = true
 
 sqlalchemy.url = ${DATABASE_URL}
 
-# secrets
-jwt.secret = ${JWT_SECRET}
+# secrets (must be set)
+jwt.secret = ${JWT_SECRET:?}
```

Comment:
Why is this needed? How does it manifest?

Reply:
It will raise a runtime error if the variable is not defined. Same as bash.
```
bash-3.2$ echo ${UNSET_VAR:?}
bash: UNSET_VAR: parameter null or not set
bash-3.2$ echo $?
1
```

Reply:
It's also possible to use a custom error message:
```bash
bash-3.2$ echo ${UNSET_VAR:?custom error message}
bash: UNSET_VAR: custom error message
```